src/messages/MOSDRepOpReply.h

```diff
@@ -102,11 +102,14 @@ class MOSDRepOpReply : public Message {
     result(result_),
     final_decode_needed(false) {
     set_tid(req->get_tid());
+    set_priority(CEPH_MSG_PRIO_HIGH);
```

Comment:
ReplicatedBackend already sets this.  Either remove it from here, or remove it from ReplicatedBackend::sub_op_modify_*